Aztec/Classes/TextKit/TextView.swift

```diff
@@ -674,17 +679,19 @@ open class TextView: UITextView {
         }
 
         let isBegginingOfListItem = storage.isStartOfNewLine(atLocation: range.location)
+        guard text == "\n" && beforeString == "\n" && afterString == "\n" && isBegginingOfListItem else {
```

Comment:
We can opt to use [this approach](https://github.com/wordpress-mobile/WordPress-Aztec-iOS/pull/209/files#diff-3660cac0e28198d6628cd6fce9c2feb9R82) as well.

Either way we should unify a solution for these character constants.